lib/travis/logs/aggregate.rb

```diff
@@ -19,15 +19,23 @@ def setup
       def run
         loop do
           aggregate_logs
-          sleep Travis.config.logs.intervals.vacuum
+          sleep sleep_interval
         end
       end
 
       def aggregate_logs
-        Travis::Logs::Services::AggregateLogs.run
+        aggregator.run
       rescue Exception => e
         Travis::Exceptions.handle(e)
       end
+
+      private def aggregator
```

Comment:
It's a style I started experimenting it and I kinda like it, as you can see from looking at the method that it's private without having to scan the entire class and don't accidentally add methods with the wrong visibility (I think that Sven indents private methods one level further for the same reason, but I personally dislike this and it makes you still scan for indentation mismatch).